### PR TITLE
Docs: Theme Changes for Decent HTML Output

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

I've tried to fix the previous light theme for the docs on sphinx but it was in vain. Set up a default RTD theme which should give us no trouble. 
+ added index files for better ToC

### Benefits

Decent HTML Layout in docs?

### Possible Drawbacks 

Not so good-looking. But when we have more time we can work this out and add another theme

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -70,25 +70,39 @@
...
+# This is required for the alabaster theme
+# refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
+#html_sidebars = {
+#    '**': [
+#        'about.html',
+#        'navigation.html',
+#        'relations.html',
+#        'searchbox.html',
+#    ]
+#}
```
Comment (eadventurous) [inline docs/source/conf.py:95]: If the theme is changed do you still need this configs for the previous theme?

Comment (LiraLemur) [reply docs/source/conf.py:95]: In the future I hope to revive a better-looking theme but you're right, in theory we can delete the contents (but better leave the html_sidebars stay there - it is a general setting used in different themes.

Comment (humb1t) [reply docs/source/conf.py:95]: @LiraLemur this code will be available in git history - so do not need to comment it. Delete and look at history when needed.

Comment (LiraLemur) [reply docs/source/conf.py:95]: I can look at it but those comments were a great help when I started doing that configuration stuff long time ago. But ok, it's not a big deal :)

**Pull request merged**